bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.airpurifier.vb2-miot.json

```
@@ -0,0 +1,1098 @@
+				"siid": 10,
+				"piid": 4,
+				"friendlyName": "Motor Speed - M1 Med L",
+				"channel": "m1_med_l",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 300,
+					"maximum": 2300,
+					"step": 10,
+					"pattern": "%.0f"
+				},
+				"refresh": true,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "NUMBER"
+					}
+				]
+			},
+			{
+				"property": "m1-low",
+				"siid": 10,
+				"piid": 5,
+				"friendlyName": "Motor Speed - M1 Low",
+				"channel": "m1_low",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 300,
+					"maximum": 2300,
+					"step": 10,
+					"pattern": "%.0f"
+				},
+				"refresh": true,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "NUMBER"
+					}
+				]
+			},
+			{
+				"property": "m1-silent",
+				"siid": 10,
+				"piid": 6,
+				"friendlyName": "Motor Speed - M1 Silent",
+				"channel": "m1_silent",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 300,
+					"maximum": 2300,
+					"step": 10,
+					"pattern": "%.0f"
+				},
+				"refresh": true,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "NUMBER"
+					}
+				]
+			},
+			{
+				"property": "m1-favorite",
+				"siid": 10,
+				"piid": 7,
+				"friendlyName": "Motor Speed - M1 Favorite",
+				"channel": "m1_favorite",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 300,
+					"maximum": 2300,
+					"step": 10,
+					"pattern": "%.0f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "motor1-speed",
+				"siid": 10,
+				"piid": 8,
+				"friendlyName": "Motor Speed - Motor1 Speed",
+				"channel": "motor1_speed",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 10000,
+					"step": 1,
+					"pattern": "%.0f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "motor1-set-speed",
+				"siid": 10,
+				"piid": 9,
+				"friendlyName": "Motor Speed - Motor1 Set Speed",
+				"channel": "motor1_set_speed",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 10000,
+					"step": 1,
+					"pattern": "%.0f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "favorite-level",
+				"siid": 10,
+				"piid": 10,
+				"friendlyName": "Motor Speed - Favorite Level",
+				"channel": "favorite_level",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 9,
+					"step": 1,
+					"pattern": "%.0f"
+				},
+				"refresh": true,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "NUMBER"
+					}
+				]
+			},
+			{
+				"property": "use-time",
+				"siid": 12,
+				"piid": 1,
+				"friendlyName": "Use Time - Use Time",
+				"channel": "use_time",
+				"type": "Number:Duration",
```

```suggestion
				"type": "Number:Time",
```
Please update elsewhere as appropriate.

---

Yep, this one also had it